tools/clang/tools/dxcompiler/dxcpdbutils.cpp

```diff
@@ -770,6 +775,8 @@ struct DxcPdbUtils : public IDxcPdbUtils, public IDxcPixDxilDebugInfoFactory
     }
     CATCH_CPP_RETURN_HRESULT()
 
+    SetEntryPointToDefaultIfEmpty();
```

Comment:
`OverrideRootSignature` could also run into the same problem. Should call it there too.